bundles/org.openhab.binding.avmfritz/README.md

```diff
@@ -8,6 +8,10 @@ The binding integrates AVM FRITZ!Boxes with a special focus on the AHA ([AVM Hom
 
 ### FRITZ!Box
 
+Attention: If you changed the Fritz!Box Name to any value that is NOT "fritz.box", the configurion in the openhab UI is not working correctly. 
```

Comment:
```suggestion
Attention: If you changed the Fritz!Box Name to any value that is NOT "fritz.box", the configuration in the openhab UI is not working correctly. 
```

Is this only an UI issue? If the thing is configured in files it is working with a different hostname?